### PR TITLE
In 131918563 saving participant

### PR DESCRIPTION
**Pull request by NateOSI**

### CODE REVIEW

File: app/repositories/participant_repository.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+# ParticipantRepository is a service class responsible for creation of a participant
+# resource via the API
+class ParticipantRepository
+  PARTICIPANTS_PATH = '/api/v1/participants'
+  CONTENT_TYPE = 'application/json'
+
+  def self.create(participant)
+    response = make_api_call(PARTICIPANTS_PATH, :post, participant)
+    raise 'Error creating participant' if response.status != 201
```
Comment (pemaOSI) [inline app/repositories/participant_repository.rb]: What happens when status is not 201?

Comment (NateOSI) [inline app/repositories/participant_repository.rb]: created a story in the ice box

Comment (pemaOSI) [inline app/repositories/participant_repository.rb]: Thats good enough.

**Comment by pemaOSI** [comment]
Close this pull request if it is done ..

**Pull request closed without merging**